<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPrefEQDlg</class>
 <widget class="QWidget" name="DlgPrefEQDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Equalizer Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="EQRackOptions">
     <property name="title">
      <string>Equalizer Rack</string>
     </property>
      <layout class="QVBoxLayout" name="Racklayout" stretch="0,0,0,0">
       <item>
        <widget class="QCheckBox" name="CheckBoxEqOnly">
         <property name="text">
          <string>Only allow EQ knobs to control EQ-specific effects</string>
         </property>
          <property name="checked">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string>Uncheck to allow any effect to be loaded into the EQ knobs.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckBoxSingleEqEffect">
         <property name="text">
          <string>Use the same EQ filter for all decks</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string>Uncheck to allow different decks to use different EQ effects.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="GridWidgetDecks" native="true">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="3">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QWidget" name="widget" native="true"/>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Equalizer Plugin</string>
            </property>
            <property name="leftMargin">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="leftMargin">
             <number>3</number>
            </property>
            <property name="text">
             <string>Quick Effect</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GroupBoxHiEQ">
     <property name="title">
      <string>High Shelf EQ</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="5">
       <widget class="QSlider" name="SliderHiEQ">
        <property name="minimum">
         <number>80</number>
        </property>
        <property name="maximum">
         <number>480</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>80</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickInterval">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="TextLabel8_3">
        <property name="font">
         <font/>
        </property>
        <property name="text">
         <string>16 Hz</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="TextLabel8_2_2">
        <property name="font">
         <font/>
        </property>
        <property name="text">
         <string>20.05 kHz</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="TextHiEQ">
        <property name="text">
         <string notr="true">textLabel1</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>SliderHiEQ</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GroupBoxLoEQ">
     <property name="title">
      <string>Low Shelf EQ</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QSlider" name="SliderLoEQ">
        <property name="minimum">
         <number>80</number>
        </property>
        <property name="maximum">
         <number>480</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>80</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickInterval">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="TextLabel8_3_2">
          <property name="font">
           <font/>
          </property>
          <property name="text">
           <string>16 Hz</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="TextLoEQ">
          <property name="text">
           <string notr="true">textLabel2</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>SliderLoEQ</cstring>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="TextLabel8_2_2_2">
          <property name="font">
           <font/>
          </property>
          <property name="text">
           <string>20.05 kHz</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="MasterMasterEQ">
     <property name="title">
      <string>Master EQ</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QComboBox" name="comboBoxMasterEq"/>
        </item>
        <item>
         <widget class="QPushButton" name="pbResetMasterEq">
          <property name="text">
           <string>Reset Parameter</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QGridLayout" name="slidersGridLayout"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="EQMiscOptions">
     <property name="title">
      <string>Miscellaneous</string>
     </property>
      <layout class="QVBoxLayout" name="miscOptionsLayout" stretch="0,0,0,0">
       <item>
        <widget class="QCheckBox" name="CheckBoxEqAutoReset">
         <property name="toolTip">
          <string>Resets the equalizers to their default values when loading a track.</string>
         </property>
         <property name="text">
          <string>Reset equalizers on track load</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckBoxGainAutoReset">
         <property name="toolTip">
          <string>Resets the deck gain to unity when loading a track.</string>
         </property>
         <property name="text">
          <string>Reset gain on track load</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckBoxBypass">
         <property name="text">
          <string>Bypass EQ effect processing</string>
         </property>
         <property name="toolTip">
          <string>When checked, EQs are not processed, improving performance on slower computers.</string>
         </property>
        </widget>
       </item>
      </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>CheckBoxEqOnly</tabstop>
  <tabstop>CheckBoxSingleEqEffect</tabstop>
  <!-- TODO(ronso0)
      Dynamically add Eq / Quick Effect comboboxes -->
  <tabstop>SliderHiEQ</tabstop>
  <tabstop>SliderLoEQ</tabstop>
  <tabstop>comboBoxMasterEq</tabstop>
  <tabstop>pbResetMasterEq</tabstop>
  <!-- TODO(ronso0)
      Dynamically add Master EQ sliders -->
  <tabstop>CheckBoxEqAutoReset</tabstop>
  <tabstop>CheckBoxGainAutoReset</tabstop>
  <tabstop>CheckBoxBypass</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
